src/components/pages/Home.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Icon } from '@iconify/react';
+import { getCryptoCoinsAsync } from '../redux/crypto/cryptoCoinSlice';
+import '../styles/Home.css';
+
+export default function Home() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCryptoCoinsAsync());
+  }, [dispatch]);
+
+  const { cryptoCoins } = useSelector((state) => state.cryptoCoins);
+
+  if (!cryptoCoins) {
+    return <p>Noo coins loaded</p>;
+  }
+
+  return (
+    <>
+      <div className="nav">
+        <h1>Touch coin</h1>
+        <div className="icons-cont">
+          <Icon icon="mdi:microphone" />
+          <Icon icon="material-symbols:settings" />
+        </div>
+      </div>
+      <section className="home-cont">
+        {cryptoCoins.map((cryptoCoin) => (
+          <article className="coin" key={cryptoCoin.id}>
+            <Link to={`touch-coin/details/${cryptoCoin.id}`}>
+              <Icon className="link" icon="mdi:arrow-right-thin-circle-outline" />
+            </Link>
+            <article className="logo-cont">
```

Comment:
- [ ] Please implement a `Filter` stateless component as a way to filter the coins using given parameters e.g coin name as per requirement 👍 
- [ ] 
![image](https://user-images.githubusercontent.com/46839548/227284222-63daa80d-1092-4059-b1aa-700bd23c96bb.png)